benchmark.py

```diff
@@ -489,6 +492,8 @@ def create_summary(data, project_source):
                     ('The platform on which bazel-bench is run. This is just '
                      'to categorize data and has no impact on the actual '
                      'script execution.'))
+flags.DEFINE_boolean('skip_clean', False, 'Whether to invoke clean between runs/builds.')
```

Comment:
Please instead make this flag `clean` and set the default value to `True`. Same for `shutdown`.

Reply:
Pushed the changes.